frontend/benefit/applicant/src/constants.ts

```diff
@@ -186,3 +186,11 @@ export const MAX_PHONE_NUMBER_LENGTH = 13 as const;
 
 // temporary consent file
 export const EMPLOYEE_CONSENT_FILE = 'employee_consent.pdf';
+
+export const SUBMITTED_STATUSES = [
+  'received',
+  'handling',
+  'cancelled',
+  'accepted',
+  'rejected',
+];
```

Comment:
`export const list = [ ... ] of const;` is preferable because no-one can change it then.

Reply:
cannot fix, as it requires a lot of changes to be readonly, and then the value is copied to the local state of the component